drop-transfer/src/ws/events.rs

```diff
@@ -73,7 +73,7 @@ impl<T: Transfer> FileEventTx<T> {
         }
 
         lock.tx
-            .send(event)
+            .send((event, utils::current_timestamp()))
```

Comment:
Oh, yeah. So `current_timestamp()` would still be used here.
This also begs for a `FileEventTxInner::emit(&self, event)` method

Reply:
Question regarding both of the `emit` refactors - do we want to return an error from those functions or not? At first i thought that it would make sense to return an error, even if we `expect` all of them on the other side, at least we get a bit more info, on which event it failed to send, but the more i look at it, the more i think that this information is not really useful, and started leaning on returning void and just having a generic `expect` for the calls. What do you think?

Reply:
Yeah, go for void.
If the expect is triggered then it would indicate the severe logic flaw, with potentially far worse consequences than controllable panic in an `emit()` function.